kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -350,7 +381,47 @@ private void translateSymbols(Map<String, Boolean> attributes, SetMultimap<KLabe
             sb.append(") : ");
             convert(prod.sort(), prod, sb);
             sb.append(" ");
-            convert(attributes, addKoreAttributes(prod, functionRules, impurities, overloads), sb, null, null);
+            Att koreAtt = addKoreAttributes(prod, functionRules, impurities, overloads);
+            convert(attributes, koreAtt, sb, null, null);
+            if (koreAtt.contains("format")) {
+              formats.append("Lbl");
+              convert(prod.klabel().get().name(), formats);
+              formats.append('\n');
+              formats.append(koreAtt.get("format"));
+              formats.append('\n');
+            }
+            if (koreAtt.contains("color")) {
+              colors.append("Lbl");
+              convert(prod.klabel().get().name(), colors);
+              colors.append('\n');
+              for (int i = 0; i < prod.items().size(); i++) {
+                if (prod.items().apply(i) instanceof Terminal) {
+                  colors.append(koreAtt.get("color"));
+                  colors.append('\n');
+                }
+              }
```

Comment:
This isn't really about symbol translation directly, and may confuse future developers about where the logic for symbol translation ends and writing the extra files begins. You could consider separating out the new functionality into a function `writeSymbolUnparsingInformation` (or something similar) and just call that method here, to make the distinction clearer.

Your call.